extensions/libraries/redcore/translation/table.php

```diff
@@ -1206,4 +1206,25 @@ public static function setInstalledTranslationTables($option, $table, $contentEl
 			$translationTableModel->save($data);
 		}
 	}
+
+	/**
+	 * Method to purge contents of a translation table
+	 *
+	 * @param   string  $tableName  name of either the translations table or the table being translated I.E. #__content or #__content__rctranslations
+	 *
+	 * @return bool
+	 */
+	public static function purgeContentElement($tableName)
```

Comment:
yeah... use $translationTables = RTranslationTable::getInstalledTranslationTables(true);
then in the loop just use this function...
RTranslationTable::purgeTables($translationTable->id);